cmd/frontend/db/repos.go

```diff
@@ -253,6 +253,12 @@ type ReposListOptions struct {
 	// OnlyArchived excludes non-archived repositories from the list.
 	OnlyArchived bool
 
+	// NoPrivate excludes private repositories from the list.
+	NoPrivate bool
+
+	// OnlyPrivate excludes public repositories from the list.
```

Comment:
nit:
```suggestion
	// OnlyPrivate excludes non-private repositories from the list.
```

For the sake of GitLab's "internal" visibility.